docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -98,18 +151,22 @@ To run this guide, make sure to be in **Implement** mode to specify the technica
 </TabItem>
 </Tabs>
 
-1. A **start event** is automatically added to the canvas. Click it to display configuration and append options.
-2. Click the rectangular **Append Task** icon to append a task.
-3. Enter a descriptive name for the task, such as `Decide what's for dinner`.
-4. Change the task type by clicking the **wrench** icon. Select **User Task**.
-5. Select the user task and click on the diamond-shaped icon to append an exclusive gateway. The gateway allows to route the process flow differently, depending on conditions.
-6. Select the gateway and append a task by clicking the task icon. Repeat it to create a second process flow. Name the tasks based on what the user decides to eat: in this case, we've named ours `Prepare chicken` and `Prepare salad`.
+1. A **start event** is automatically added to the canvas and selected.
+2. Append a task. For example, hover over the blue plus icon and select the **Append Task** icon from the [context pad](/components/modeler/web-modeler/context-pad.md).
```

Comment:
Context pad is more useful for the SaaS/web modeler side of things and not the same experience in desktop. If this section is swapped to `Append element` instead (and the context pad reference removed), it brings it back in line with the Desktop experience and would remove the need for another set of split/tabbed content.